reactor-netty-http/src/main/java/reactor/netty/http/server/HttpServerOperations.java

```diff
@@ -757,7 +761,7 @@ public HttpServerResponse compression(boolean compress) {
 			removeHandler(NettyPipeline.CompressionHandler);
 		}
 		else if (channel().pipeline().get(NettyPipeline.CompressionHandler) == null) {
-			SimpleCompressionHandler handler = new SimpleCompressionHandler();
+			SimpleCompressionHandler handler = SimpleCompressionHandler.create(compressionSettings);
```

Comment:
Applies the user-defined `compression settings` to the Netty's compression settings.